### DISCUSSION TITLE
novelWriter features at Linuxcapable.com

### DISCUSSION THREAD

**Discussion by johnblommers** [vkbo/novelWriter#1454] [General]
It's always good to see novelWriter getting press. On May 30, 2023 we have:

**[How to Install novelWriter on Ubuntu 22.04 | 20.04](https://www.linuxcapable.com/how-to-install-novelwriter-on-ubuntu-linux/)**

Sweet! Congratulations &rarr; Veronica.

**Comment by vkbo** [comment]
Oh, that's nice 😄
